compiler/cpp/src/thrift/generate/t_netstd_generator.cc

```diff
@@ -3110,6 +3013,5 @@ THRIFT_REGISTER_GENERATOR(
     "C#",
     "    wcf:             Adds bindings for WCF to generated classes.\n"
     "    serial:          Add serialization support to generated classes.\n"
-    //"    nullable:        Use nullable types for properties.\n"
     "    union:           Use new union typing, which includes a static read function for union types.\n"
```

Comment:
Curious, are we going to make some of these options defaults "true" (like union) for netstd as they are improvements?